docs/reference/errors-and-warnings/NU1011.md

```diff
@@ -15,7 +15,9 @@ f1_keywords:
 
 ### Issue
 
-PackageVersion items cannot contain floating versions.
+`<PackageVersion />` items cannot contain floating versions.  NuGet's central package management (CPM) is considered an enterprise-level feature which provides easier version management at scale as well as deterministic and secure restores.  The use of floating versions introduces the possibility for a bad package to be introduced into your build after it has been pushed to a feed.  This can lead to a situation where you made no changes in your repository but suddenly something is broken and there is no way for you to get back into a good state without removing the floating version or pushing a newer version of the package which is fixed.  Using non-floating versions means that every upgrade to a package is backed by a commit in your repository making it easy to determine what change caused the break and to revert a commit to get back into a good state.

```

Comment:
Should we point out that transitive pinning also makes no sense with floating versions? 
That's also a pretty big factor, as with floating versions only, there's the lock files argument.